tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -12365,6 +12510,16 @@ void Sema::CheckHLSLFunctionCall(FunctionDecl *FDecl, CallExpr *TheCall) {
 
   hlsl::IntrinsicOp opCode = (hlsl::IntrinsicOp)IntrinsicAttr->getOpcode();
   switch (opCode) {
+  case hlsl::IntrinsicOp::MOP_AlignedLoad:
+  case hlsl::IntrinsicOp::MOP_AlignedStore:
+    // AlignedLoad/AlignedStore require SM 6.2+ (DXIL 1.2+) for
```

Comment:
This is probably okay, but I do just want to call out that drivers have never been tested seeing an alignment value here that doesn't match the size of the data being loaded, as far as I know. There was a while where WARP used the alignment value to determine the data size instead of looking at the LLVM type (I don't know why...). If we do a spot check of modern drivers across the ecosystem and they seem okay with larger alignment values, I'm fine keeping this as a retro-enablement for SM6.2, but we should do the due diligence before lighting this up for customers.